src/examples/sparqlRDFGraph.ts

```diff
@@ -0,0 +1,103 @@
+import { createElement, ClassAttributes } from 'react';
+import * as ReactDOM from 'react-dom';
+
+import { Workspace, WorkspaceProps, SparqlDataProvider, GraphBuilder } from '../../index';
+
+require('jointjs/css/layout.css');
+require('jointjs/css/themes/default.css');
+
+document.addEventListener('DOMContentLoaded', () => {
+    const container = document.createElement('div');
+    container.id = 'root';
+    document.body.appendChild(container);
+
+    const props: WorkspaceProps & ClassAttributes<Workspace> = {
+        onSaveDiagram: workspace => {
+            const layout = workspace.getModel().exportLayout();
+            console.log(layout);
+        },
+        ref: workspace => {
+            // if you reuse this code you should check for workspace to be null on unmount
+            if (workspace) {
+                const model = workspace.getModel();
+                const endpointUrl = '/sparql-endpoint';
+                const sparqlDataProvider = new SparqlDataProvider({
+                    endpointUrl: endpointUrl,
+                    imageClassUris: ['http://collection.britishmuseum.org/id/ontology/PX_has_main_representation']
+                });
+                const graphBuilder = new GraphBuilder(sparqlDataProvider, endpointUrl);
+
+                let graph = [
```

Comment:
It would be more readable if you declare GRAPH constant before `document.addEventListener(...` line.